assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BEZIER } from '../../../../animation';
+import {
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  BUTTON_SIZES,
+  Icons,
+  NotificationBubble,
+  Button as dsButton,
+} from '../../../../design-system';
+
+const Button = styled(dsButton)`
+  border: ${({ theme }) => `1px solid ${theme.colors.bg.tertiary}`};
```

Comment:
The button component already has a border value defined, so you could just set the color. 

I have a question out about how we want to handle the shift from the standard border to the 2px wide one that is present on focus for WCAG standards - so if we keep the value in the Button it'll be easier to make that update happen with whatever the choice is.